app/build.gradle

```diff
@@ -110,16 +110,6 @@ android {
         klarX86Debug.root = 'src/klarDebug'
         klarX86Debug.root = 'src/klarDebug'
         klarX86Debug.root = 'src/klarDebug'
-
-        // UI test
-        androidTestWebviewCompile {
```

Comment:
Is it okay to remove these without replacing them?

Reply:
These sourceSets were there to handle separate AndroidTest instrumentation roots for Webview and GeckoView. No-Jun already migrated all of those tests to the androidTest directory, where they belong in a default Android app root. He handled the dependency on each rendering engine by adding code that ignores the test if it's for the other engine.